p5/visualTests/Instructions.md

```diff
@@ -0,0 +1,12 @@
+# Run the visual test
+```
+python sanity_testing.py
+```
+
+**NOTE**: If you get a `ModuleNotFound or ImportError` make sure you export the `project's root path` to `PYTHONPATH` variable.
+
+```
+export PYTHONPATH=$PYTHONPATH:path/to/project"
+```
```

Comment:
@ziyaointl  Are you familiar with any other way to export the project's root path to the PYTHONPATH environment variable?
I was getting a ModuleNotFound error on running scripts through the terminal, and scripts were running fine if I was using the IDE's command to run them `Ctrl + shift + F10` in pycharm as the IDE automatically adds the project's root path to the variable.

Reply:
I prefer `pip install .` if I'm in a virtual env since it's a lot easier to type and it isn't a big deal to mess up the installed packages. Furthermore, this makes sure that an pre-existing p5 package won't interfere with the new changes we are trying to test. Maybe we should use this instead?

Reply:
Worked. Thanks!